choreolib/src/main/java/choreo/auto/AutoTrajectory.java

```diff
@@ -487,32 +466,10 @@ public Trigger atTime(String eventName) {
     return trig;
   }
 
-  private Trigger atPose(
-      Supplier<Optional<Pose2d>> pose, double toleranceMeters, double toleranceRadians) {
-    return new Trigger(
-            routine.loop(),
-            () -> {
-              Optional<Pose2d> checkedPoseOpt = pose.get();
-              return checkedPoseOpt
-                  .map(
-                      (checkedPose) -> {
-                        Translation2d currentTrans = poseSupplier.get().getTranslation();
-                        Rotation2d currentRot = poseSupplier.get().getRotation();
-                        return currentTrans.getDistance(checkedPose.getTranslation())
-                                < toleranceMeters
-                            && Math.abs(currentRot.minus(checkedPose.getRotation()).getRadians())
-                                < toleranceRadians;
-                      })
-                  .orElse(false);
-            })
-        .and(active());
-  }
-
-  private Trigger atPose(Optional<Pose2d> pose, double toleranceMeters, double toleranceRadians) {
-    return atPose(
-        AllianceFlipUtil.optionalFlippedPose2d(pose, alliance, useAllianceFlipping),
-        toleranceMeters,
-        toleranceRadians);
+  private boolean withinTolerance(Rotation2d lhs, Rotation2d rhs, double toleranceRadians) {
+    double dot = lhs.getCos() * rhs.getCos() + lhs.getSin() * rhs.getSin();
+    double sinTolerance = Math.sin(toleranceRadians);
```

Comment:
This doesn't seem right; `a dot b = ||a|| ||b|| cos theta`, not sin. In TrajoptLib we checked that the dot of the unit vectors was greater (closer to 1) than the cosine of the tolerance.